dask/array/tests/test_fft.py

```diff
@@ -195,3 +195,9 @@ def test_wrap_fftns(modname, funcname, dtype):
         wfunc(darr2c, (darr2c.shape[0] - 1, darr2c.shape[1] - 1), (0, 1)),
         func(nparrc, (nparrc.shape[0] - 1, nparrc.shape[1] - 1), (0, 1))
     )
+
+
+@pytest.mark.parametrize("n", list(range(1, 10)))
+@pytest.mark.parametrize("d", [1.0, 0.5, 2 * np.pi])
```

Comment:
I like parametrize, but only for things that are big changes (e.g. the `tasks`/`disk` parameters in the shuffle tests), but don't for things where we end up with 100s of tests for each single test function. My issue with them is mostly the increase in printed output by py.test, which isn't that big of a deal. In this case I could go either way, up to you.